src/sorunlib/wiregrid.py

```diff
@@ -305,3 +384,116 @@ def calibrate(continuous=False, elevation_check=True, boresight_check=True,
     finally:
         # Stop SMuRF streams
         run.smurf.stream('off')
+
+
+def time_constant(num_repeats=1):
```

Comment:
Probably, we should `num_repeast=2` not to change the HWP direction in the default value.